### PR TITLE
pdostal change maintainer

### PR DESCRIPTION
**Pull request by pdostal**
- Transfer ownership of virtualization tests from me, @grisu48 and @baierjan to virtualization team
- Transfer ownership of NFS4, Apache2 and Salt from me to QE Core
- Transfer ownership of Public Cloud tests to qe-c team
- Get rid of ownership of remote_desktop/screensharing_available test
    I unfortunately don't know who to transfer it to.
- I stay maintainer for Ansible, Wireguard, openVPN, SSHD, SLP, SSHd, RPM, NginX

### CODE REVIEW

**Review by grisu48 — APPROVED** [review]
LGTM, but there are CI issues.

**Review by baierjan — APPROVED** [review]

**Review by alice-suse — APPROVED** [review]
LGTM

File: tests/x11/remote_desktop/screensharing_available.pm [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # - Check if wayland is being used
 #   - If wayland is detected, record_soft_failure (boo#1137569)
 # - Finish with ctrl-q
-# Maintainer: Pavel Dostál <pdostal@suse.cz>
+# Maintainer:
```
Comment (foursixnine) [inline tests/x11/remote_desktop/screensharing_available.pm:15]: this would be Desktop team, but qe-core can be it for now

Comment (foursixnine) [reply tests/x11/remote_desktop/screensharing_available.pm:15]: https://github.com/os-autoinst/os-autoinst-distri-opensuse/actions/runs/4499546432/jobs/7917547747?pr=16691#step:6:83

**Comment by foursixnine** [comment]
> LGTM, but there are CI issues.

Captured it: https://progress.opensuse.org/issues/126518

File: tests/x11/remote_desktop/screensharing_available.pm (hunk 1 above)
Comment (pdostal) [reply tests/x11/remote_desktop/screensharing_available.pm:15]: Do you have their e-mail and some github nickname to contact for approval?

Comment (pdostal) [reply tests/x11/remote_desktop/screensharing_available.pm]: It is QE Core for now. You may merge.

**Pull request merged**